scenes/menu/preferences_window.gd

```diff
@@ -0,0 +1,25 @@
+extends PopupPanel
+
+@onready var timer_selection = $VBoxContainer/StartingTimersSelection
+@onready var best_of_selection = $VBoxContainer/BestOfSelection
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	timer_selection.select(timer_selection.get_item_index(GlobalSettings.CustomStartingTimer))
+	best_of_selection.select(best_of_selection.get_item_index((GlobalSettings.CustomStartingBestOf)))
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(_delta):
+	pass
+
+func _on_bgm_check_box_toggled(button_pressed):
+	GlobalSettings.set_bgm(button_pressed)
```

Comment:
this is where you probably want to also emit another signal so MainMenu can listen to this window instead of the button directly.